mlir/utils/jenkins/parameterSweeps.py

```diff
@@ -407,23 +413,44 @@ def main() -> bool:
         help="The build directory of MIOpen",
     )
     args = parser.parse_args()
-    options = Options(debug=args.debug, quiet=args.quiet, xdlops=args.xdlops,
-        arch=','.join(getArch()), concurrent_tests=args.jobs)
+
+    arch = ','.join(getArch())
+    supported_codepath = ['mfma', 'vanilla']
+    # If codepath not provided or not supported, infer it from the arch
+    codepath = args.codepath
+    rocmlir_gen_flags = ''
```

Comment:
You've still got the weirdness for rocmlir_gen_flags in here too

Reply:
So you don't need the flag for `parameterSweeps.py` or you also want `rocmlir_gen_flags` as a command line option besides `codepath`?

Reply:
Ah, I got thrown by the name.

The codepath-based method is fine, I don't think anyone will need any parameter sweeps weirder than that.

I'd go ahead and make `rocmlir_gen_flags` a list in this version so you don't have to `split()` it every time, but that's nitpicking and won't block the PR